css/ot_drawer.css

```diff
@@ -52,7 +52,7 @@ html.ot-drawer-open, html.ot-drawer-open > body {
 .ot-drawer {
 
     position: fixed;
-    top: 0;
+    width: 100%;
```

Comment:
Do we really want to force a width of `100%`?
(On https://ocsigen.org/ocsigen-start/, the drawer does not cover the whole page.)